table/tables/index_test.go

```diff
@@ -122,7 +130,7 @@ func (s *testIndexSuite) TestIndex(c *C) {
 
 	_, hit, err := index.Seek(sc, txn, values)
 	c.Assert(err, IsNil)
-	c.Assert(hit, IsTrue)
+	c.Assert(hit, IsFalse)
```

Comment:
Why change this?

Reply:
The original implementation uses 0 as int handle, so the `hit` is true if the handle happens to be 0.
The new implementation doesn't append anything to the index key, so the `hit` is false.

The hit is only used in test code, so it doesn't matter.